.circleci/config.yml

```diff
@@ -8,6 +8,7 @@ default_docker: &default_docker
     DATAGOUVFR_SITE: https://demo.data.gouv.fr
     MAILCHIMP_NEWSLETTER_URL: //gouv.us13.list-manage.com/subscribe/post?u=5ee8bfe0f1b073b49de06a063&amp;id=13db2e9a94
     GTFS_VALIDATOR_URL: https://transport-validator.cleverapps.io
+    SECRET_KEY_BASE: "hJbD4eqmDrNX+/LjUGMLQWQDRMYSr2o+hlIEzECdmPrNHCWmZtXCO879MUU3YcEz"
```

Comment:
Voir plus bas, le code fait à présent un `fetch_env!` pour l'env de test, ce qui va échouer si on ne met pas une valeur.

Reply:
mais c'est pas un vrai secret ?

Reply:
non je l'ai généré exprès mais c'est une bonne remarque, je vais expliciter et mettre un texte dedans si ça passe pour que ça enlève ce doute

Reply:
J'ai remplacé par une chaîne de caractère `SOME-SECRET-KEY-BASE-FOR-TESTING-SOME-SECRET-KEY-BASE-FOR-TESTING` comme ça c'est clair que c'est pas un credential.

Merci pour la remarque !